plugins/xlsx-worbook-sink/src/plugin.spec.ts

```diff
@@ -0,0 +1 @@
+
```

Comment:
No easy way to test this. I could use cypress or playwright to write a full e2e test if that's what we want.